contracts/FlashRollover.sol

```diff
@@ -0,0 +1,344 @@
+pragma solidity ^0.8.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+import "./external/interfaces/ILendingPool.sol";
+
+import "./interfaces/IFlashRollover.sol";
+import "./interfaces/ILoanCore.sol";
+import "./interfaces/IOriginationController.sol";
+import "./interfaces/IRepaymentController.sol";
+import "./interfaces/IAssetWrapper.sol";
+import "./interfaces/IFeeController.sol";
+
+contract FlashRollover is IFlashRollover {
```

Comment:
Yeah my opinion on that is that I don't like `solidity-docgen`, it's not flexible enough formatting-wise, so I hand rolled a doc. Do you think it would suffice to just link it here in a comment above the entire contract?

Reply:
Here's example for cusotmizing formatting of solidity-docgen https://forum.openzeppelin.com/t/incorporating-solidity-docgen-into-your-project/1882

but what you have here is fine